mitodl/ocw-studio
# documentation of adding CKEditor plugin

#### What are the relevant tickets?

part of #39 

#### What's this PR do?

I wrote up documentation of implementing a CKEditor plugin to add YouTube embeds from scratch. We don't actually want to use such a plugin ourselves (we'd want to just use CKEditor's media embed plugin) but it is helpful as an example of how to build a plugin in general.

#### How should this be manually tested?

Read it and make sure it all makes sense.

If you wanted to be really adventurous you could try to follow along!
